TechGuru/settings.py

```diff
@@ -53,13 +62,16 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+AUTHENTICATION_BACKENDS = [
```

Comment:
Why you added this? Its already done by default.

https://docs.djangoproject.com/en/5.0/ref/settings/#authentication-backends